static/js/timerender.js

```diff
@@ -171,7 +171,8 @@ exports.render_markdown_timestamp = function (time, now, text) {
         now = now.tz(page_params.timezone);
         time = time.tz(page_params.timezone);
     }
-    const timestring = time.format('ddd, MMM D YYYY, h:mm A');
+    const hourformat = page_params.twenty_four_hour_time ? 'HH:mm' : 'h:mm A';
+    const timestring = time.format('ddd, MMM D YYYY, ' + hourformat);
```

Comment:
Actually, better solution that realistically doesn't need a test: let's call `exports.stringify_time` here.

Reply:
Oh, hmm, that likely won't work due to library differences.